src/handlers/thirdparty.ts

```diff
@@ -185,3 +183,26 @@ export function handleItemAdded(event: ItemAdded): void {
   let metric = buildCountFromItem()
   metric.save()
 }
+
+export function handleItemReviewed(event: ItemReviewed): void {
+  let itemId = buildItemId(event.params._thirdPartyId, event.params._itemId)
```

Comment:
There's a check we're doing when adding a new item that might be useful:
```
  if (!isBlockchainIdValid(event.params._itemId)) {
    log.error('An item was added in the TPR "{}" with an incorrect id "{}"', [
      event.params._thirdPartyId,
      event.params._itemId
    ])
    return
  }
```

Reply:
Oh! nice